### DISCUSSION TITLE
Cash Flow data problems, with proceedsFromRepaymentsOfShortTermDebt Value

### DISCUSSION THREAD

**Discussion by hamdi369** [xrgarcia/alphavantage_api_client#6] [Q&A] [answered]
Hi, I am Hamdi Fouzai, french, 44 years old. I am trying to create a windows application using clickfusion 2.5. It's first time I discuss here. I found no official Alpha Vantage discussion forum...

I am currently working on Fundamantal Data API CASH FLOW part.

I noticed an inconsistency:

when testing on **_AAPL_** the value _**"proceedsFromRepaymentsOfShortTermDebt"**_ is doubled. It wouldn't have been a problem if this value is doubled on all stocks. I would just have to divide by 2 to correct it, but when tested on **_MSFT_**, this value is correct and does not need correction.

It is impossible to discriminate (make difference) between values ​​that need corrections (as in AAPL) and others that do not (as in MSFT)

Does anyone know about this problem and found a solution?

[Alpha-Vantage_Cash_Flow_problem.pdf](https://github.com/xrgarcia/alphavantage-api-client/files/8845746/Alpha-Vantage_Cash_Flow_problem.pdf)

**Comment by xrgarcia** [comment, marked as answer]
Hi Hamdi, It looks like a string value, so i wouldn't expect .NET to mess with the value as it might with a float or a double.  I recommend you DM or tweet at @alpha_vantage to get support.
